pkg/kubectl/cmd/cp.go

```diff
@@ -184,6 +216,20 @@ func copyToPod(f cmdutil.Factory, cmd *cobra.Command, stdout, stderr io.Writer,
 	}()
 
 	// TODO: Improve error messages by first testing if 'tar' is present in the container?
+	if err := testTar(f, cmd, dest); err != nil {
+
+		exitCode126 := "command terminated with exit code 126"
+		exitCode127 := "command terminated with exit code 127"
+
+		if err.Error() == exitCode126 {
```

Comment:
move this to testTar to remove duplication at both places?

Reply:
will do! Somehow forgot about that!